geniza/corpus/admin.py

```diff
@@ -283,11 +283,13 @@ def queryset(self, request, queryset):
 
             # use Solr to take advantage of processed date range fields
```

Comment:
I find it hard to read the or logic inside the queryset filter, can you set the field before you create and filter the queryset instead?

```suggestion
            # use Solr to take advantage of processed date range fields
            date_field = 'document_dated_dr' if request.GET.get("exclude_inferred", None) == "true" else 'document_dating_dr'
```
then you can set date filter options like this `date_filter_opts = {date_field: date_filter}` and pass them to the filter as kwargs `.filter(**date_filter_opts)`


Or maybe I'm misreading this because of the diff. Simpler might be to put the if/else at the outer level and just initialize the `sqs` two different ways